src/BookBundle/Controller/ProjectController.php

```diff
@@ -110,10 +110,28 @@ public function showAction(Project $project)
     {
         $deleteForm = $this->createDeleteForm($project);
 
-        return $this->render('project/show.html.twig', array(
-            'project' => $project,
-            'delete_form' => $deleteForm->createView(),
-        ));
+        if (in_array('ROLE_ADMIN', $this->getUser()->getRoles())) {
+            return $this->render('project/show.html.twig', array(
```

Comment:
un return en trop ! 
tu fais exactement le meme return $this->render dans le else qui suit, ce n'est pas DRY !